resources/views/admin/speaker/view.twig

```diff
@@ -15,6 +15,15 @@
             <div>
                 {% if speaker.company %}<span class="mr-3"><i class="fa fa-building"></i> {{ speaker.company }}</span>{% endif %}
                 {% if speaker.twitter %}<span class="mr-3"><a class="hover:text-brand" target="_blank" href="https://twitter.com/{{ speaker.twitter }}" rel="noopener noreferrer"><i class="fa fa-twitter"></i> @{{ speaker.twitter }}</a></span>{% endif %}
+                {% if speaker.getJoindInUsername() %}
+                    <span class="mr-3">
+                        <a class="hover:text-brand" target="_blank"
+                           href="https://joind.in/user/{{ speaker.getJoindInUsername() }}"
```

Comment:
Unless I am terribly mistaken, and to answer your question from the body of the pull request, I think it should be possible to use 

```diff
diff --git a/resources/views/admin/speaker/view.twig b/resources/views/admin/speaker/view.twig
index ace150ce..3d9e23fe 100644
--- a/resources/views/admin/speaker/view.twig
+++ b/resources/views/admin/speaker/view.twig
@@ -18,9 +18,9 @@
                 {% if speaker.getJoindInUsername() %}
                     <span class="mr-3">
                         <a class="hover:text-brand" target="_blank"
-                           href="https://joind.in/user/{{ speaker.getJoindInUsername() }}"
+                           href="https://joind.in/user/{{ speaker.joindInUsername }}"
                            rel="noopener noreferrer">
-                            <i class="fa fa-commenting-o"></i> {{ speaker.getJoindInUsername() }}
+                            <i class="fa fa-commenting-o"></i> {{ speaker.joindInUsername }}
                         </a>
                     </span>
                 {% endif %}
```

here instead. Have you tried?

Maybe it also makes sense to extract a method and shift the construction of the URL to `SpeakerProfile`, what do you think?

Reply:
I like the idea of having `SpeakerProfile` generate this too since I recently merged in a PR that moved the generation of Twitter URL's to `SpeakerProfile` as well

Reply:
Haha, I spent half an hour on stream trying to figure out why `speaker.joindInUsername` hadn't been working :p I can certainly give it another crack tonight though!

I can definitely be on board with either constructing the URL through SpeakerProfile or on a JoindIn helper (and something similar for Twitter as well)

Reply:
Ah, well then, that answers the second part!